packages/TorneloScoresheet/src/pages/TablePairingSelection.tsx

```diff
@@ -1,106 +1,174 @@
-  const [appModeState, {returnToPgnLinkPage}] = useAppModeState();
-  
-  if (appModeState.mode !== AppMode.TablePairing) {
+  const [appModeState, { goToEnterPgnLink, goToTablePairingMode }] = useAppModeState();
+  const [showConfirmButton, setShowConfirm] = useState(true);
+  const [selectedPairing, setSelected] = useState<GameInfo|null>(null);
+  if (appModeState.mode !== AppMode.ArbiterSetup) {
+    return <></>;
+  }
+  if (!appModeState.pairings) {
     return <></>;
   }
+  const onSelectPairing = (pairing: GameInfo) =>{
+    if (selectedPairing === pairing){
+      setSelected(null)
+      setShowConfirm(false)
 
+    }else{
+      setSelected(pairing)
+      setShowConfirm(true)
+    }
+  }
+  const paringCardStyle = (pairing: GameInfo) => {
+    if (selectedPairing === pairing){
+      return ({
+        backgroundColor: colours.primary
+      })
+    }else{
+      return ({
+        backgroundColor: colours.secondary,
 
-  const renderPairing = ({ item }: { item: GameInfo }) => (
-    <View style={styles.pairingCard}>
-        <View style={styles.roundTextSection}>
-            <Text style={styles.roundText}>{item.round}.{item.subRound}</Text>
-        </View>
-        <View style={styles.nameTextInnerSection}>
-            <Text style={styles.nameText}>{item.players[0].firstName} {item.players[0].lastName}</Text>
-            <Text style={styles.nameText}>{item.players[1].firstName} {item.players[1].lastName}</Text>
-        </View>
-    </View>)
+      })
+    }
+  }
+  
 
+  const renderPairing = ({ item }: { item: GameInfo }) => (
+    <TouchableOpacity style={[styles.pairingCard, paringCardStyle(item)]} onPress={() => {onSelectPairing(item)}}>
+      <View style={styles.roundTextSection}>
+        <Text style={styles.roundText}>
+          {item.round}.{item.subRound}
+        </Text>
+      </View>
+      <View style={styles.nameTextInnerSection}>
+        <Text style={styles.nameText}>
+          {item.players[0].firstName} {item.players[0].lastName}
+        </Text>
+        <Text style={styles.nameText}>
+          {item.players[1].firstName} {item.players[1].lastName}
+        </Text>
+      </View>
+    </TouchableOpacity>
+  );
+  const renderConfirm = () => {
+    if(showConfirmButton && selectedPairing !== null){
+    return(<Text  onPress={() => goToTablePairingMode(selectedPairing)} style={styles.forwardBtn}>
+      Confirm {'>'}
+    </Text>)
+    }
+  }
+  
   return (
-  <View>
-      <Text onPress={returnToPgnLinkPage} style={styles.backBtn}>{'<'} Back</Text>
-      <Text style={styles.instructionTitle}>Boards</Text>
-      <Text style={styles.instructionContents}>Select the board that this iPad is assigned to. This can be changed later.</Text>
-      <FlatList scrollEnabled={true} style={styles.pairingList} data={appModeState.pairings} renderItem={renderPairing}>
+    <View>
+      <View style={styles.buttonContainer}>
+        <Text onPress={goToEnterPgnLink} style={styles.backBtn}>
+          {'<'} Back
```

Comment:
Let's use a chevron from the "react-native-vector-icons" we have installed here? Don't necessarily have to this in this PR, since this PR isn't a final iteration of the UI, just wanted to flag it

Reply:
yeh just not bothering since the desing might change, well change it later